program/rust/src/tests/test_add_publisher.rs

```diff
@@ -57,10 +56,14 @@ fn test_add_publisher() {
         Err(OracleError::InvalidSignableAccount.into())
     );
 
+    #[cfg(not(feature = "pythnet"))]
```

Comment:
Since we re-adjust lamports manually in a later line, we tell this test case what the size should be conditionally.